frontend/src/pages/dashboard/components/table/drivers-table/columns/columns.ts

```diff
@@ -0,0 +1,32 @@
+import { type ColumnDef } from '@tanstack/react-table';
+
+import { type DriverWithUserData } from '~/libs/types/types.js';
+
+const columns: ColumnDef<DriverWithUserData>[] = [
+  {
+    header: 'Full Name',
+    accessorFn: (driver) => `${driver.firstName} ${driver.lastName}`,
```

Comment:
I believe we have a helper for this, not sure it's in dev though.

Reply:
I did find helpers for it in dev, should I write helper for it?